addons/im_livechat/static/src/models/messaging_initializer.js

```diff
@@ -8,17 +8,14 @@ import '@mail/models/messaging_initializer';
 patchRecordMethods('MessagingInitializer', {
     /**
      * @override
-     * @param {Object} resUsersSettings
-     * @param {boolean} resUsersSettings.is_discuss_sidebar_category_livechat_open
      */
-    _initResUsersSettings({ is_discuss_sidebar_category_livechat_open }) {
-        this._super(...arguments);
-        this.messaging.userSetting.update({ is_discuss_sidebar_category_livechat_open });
+    _initResUsersSettings(...args) {
         this.messaging.discuss.update({
             categoryLivechat: insertAndReplace({
                 serverStateKey: 'is_discuss_sidebar_category_livechat_open',
```

Comment:
This should be a compute too, why isn't it removed in the commit doing the others?